pkgs/development/tools/build-managers/qbs/default.nix

```diff
@@ -3,13 +3,13 @@
 stdenv.mkDerivation rec {
   name = "qbs-${version}";
 
-  version = "1.8";
+  version = "1.12";
 
   src = fetchFromGitHub {
     owner = "qt-labs";
```

Comment:
Apparently the owner is now `qbs` since github.com/qt-labs/qts redirects to github.com/qts/qts.